nucleus/common/common-util/src/main/java/com/sun/enterprise/universal/security/SecurityUtils.java

```diff
@@ -48,9 +48,9 @@
  * @author Byron Nevins
  */
 public final class SecurityUtils {
+    private static SecureRandom random = new SecureRandom();
 
     public static String getSecureRandomHexString(int numBytes) {
-        SecureRandom random = new SecureRandom();
         byte[] bb = new byte[numBytes];
         random.nextBytes(bb);
```

Comment:
Use `RandomUtils` here to consolidate the `SecureRandom` usage into a single utility class to maintain consistency and manage randomness more effectively.

Reply:
Should be fixed separately as may cause OSGI issue, due to package naming.